nightly/pytest-sanity.txt

```diff
@@ -217,3 +222,6 @@ pytest --timeout=120 sanity/chunk_validator_failover.py --features nightly
 # TODO(#13296): Enable after fixing flaky test
 # pytest --timeout=240 sanity/epoch_sync.py
 # pytest --timeout=240 sanity/epoch_sync.py --features nightly
+
+# Very expensive test: make sure Docker image can be build and run
+pytest --skip-build --timeout=1h sanity/docker.py
```

Comment:
Unrelated to this PR - can we make the slow tests to be run first? It may speed things up in our multi worker setup.

Reply:
Yes, I did think of doing that but I don't think it's super straight forward. The code for querying tests is here: https://github.com/Near-One/nayduck/blob/d7269e1825a363f125021672088e68bb852e9767/workers/worker_db.py#L33-L56

and we would probably need to add some clause to order by avg test execution time